program/src/processor.rs

```diff
@@ -435,7 +435,7 @@ impl Processor {
         }
 
         // Withdrawal period is 7 days = 7 * 86400 = 604_800
-        schedule.release_time = clock.unix_timestamp as u64 + 604_800;
+        schedule.release_time = clock.unix_timestamp as u64 + 1800;
```

Comment:
- pre-release version, so we can test withdrawals quickly
- production will have the month buffer